.gitattributes

```diff
@@ -0,0 +1,3 @@
+*.go	text eol=lf whitespace=trailing-space,space-before-tab,indent-with-non-tab diff=golang
```

Comment:
what's the point of whitespace=...? gofmt already does all of it automatically.
same for go.mod/sum which are always auto generated and updated.

Reply:
Yes, `gofmt` formats it the right way.
This emits warnings when staging files that may be accidentally incorrectly formatted.